src/fann.c

```diff
@@ -683,13 +683,20 @@ FANN_EXTERNAL fann_type *FANN_API fann_run(struct fann *ann, fann_type *input) {
           neuron_it->value = neuron_sum;
           break;
         case FANN_LINEAR_PIECE:
-          neuron_it->value = (fann_type)(
-              (neuron_sum < 0) ? 0 : (neuron_sum > multiplier) ? multiplier : neuron_sum);
+          neuron_it->value = (fann_type)((neuron_sum < 0)            ? 0
+                                         : (neuron_sum > multiplier) ? multiplier
+                                                                     : neuron_sum);
```

Comment:
NIT: Formatting should actually be done using `./format.sh` script that uses clang-format so this will get overwritten. I will run it after merging.

Reply:
I should really add check for that to the pipeline.